modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -1,20 +1,26 @@
 // Module included in the following assemblies:
 //
-// scalability_and_performance/ztp-advanced-policy-config.adoc
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
 
 :_module-type: PROCEDURE
 [id="ztp-configuring-pgt-image-registry_{context}"]
 = Configuring the image registry using PolicyGenTemplate CRs
 
-You can use `PolicyGenTemplate` to apply to create the PV and PVC and patch `imageregistry` configuration. Select the appropriate `PolicyGenTemplate` for each `source-cr`. See Additional Resources for more help.
+Use `PolicyGenTemplate` (PGT) CRs to apply the CRs required to configure the image registry and patch the `imageregistry` configuration.
 
 .Prerequisites
 
-* You have installed and configured Zero Touch Provisioning (ZTP). For information about this, see the topic on ZTP in Additional resources.
+* You have configured a disk partition in the managed cluster that you provisioned with GitOps ZTP. See "Configuring disk partitioning with SiteConfig".
```

Comment:
It's not a must to provision disk partition with ZTP. Using SiteConfig is just a convenient  way of doing that.

Reply:
updated